examples/patient-hub/flows/Admissions.flow.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name" : "Admissions",
+  "description" : "",
+  "batchSize" : 100,
+  "threadCount" : 4,
+  "stopOnError" : false,
+  "options" : { },
+  "version" : 0,
+  "steps" : {
+    "1" : {
+      "name" : "AdmissionsCore",
+      "description" : "",
+      "options" : {
+        "additionalCollections" : [ ],
+        "headers" : {
+          "sources" : [ {
+            "name" : "Admissions"
+          } ],
+          "createdOn" : "currentDateTime",
+          "createdBy" : "currentUser"
+        },
+        "sourceQuery" : "cts.collectionQuery([])",
+        "collections" : [ "AdmissionsCore" ],
+        "permissions" : "rest-reader,read,rest-writer,update",
+        "outputFormat" : "json",
+        "targetDatabase" : "data-hub-STAGING"
+      },
+      "customHook" : { },
+      "retryLimit" : 0,
+      "batchSize" : 0,
+      "threadCount" : 0,
+      "stepDefinitionName" : "default-ingestion",
+      "stepDefinitionType" : "INGESTION",
+      "fileLocations" : {
+        "inputFilePath" : "/PATH/TO/datasets/admissions-core",
```

Comment:
I noticed that if this is set to "datasets/admissions-core", then it will resolve to the correct path when the user loads the project. I think this is worth doing, as it's fairly tedious to use the file browser in the ingest step to find the correct path, it can take many clicks to achieve that.